### ISSUE TITLE
PFS provides disjunct paths

### ISSUE THREAD

**Issue by christianbrb** [raiden-network/raiden-pathfinding-service#87]
## Description

**As Josh I want to be able to**

**So that I can**

## Acceptance Criteria

- [ ] 

## Related story / issue
- 

## Reviewer
-

**Comment by karlb** [comment]
> One can set "disjunctive path" as a argument at request with the PFS API

Do we want to make the disjunctiveness configurable? Our current spec says it's hardcoded.

If we want to add an argument, I suggest

> `channel_overlap_penalty`: For each channel, a route will receive `channel_overlap_penalty * occurrences_of_channel_in_previous_routes` penalty points.
For comparison: one penalty point the penalty for increasing the route length by one node/channel. So if `channel_overlap_penalty = 10` reusing a channel once is as bad as adding ten new nodes to the route.

But I think it's best to skip that for now and just use a hard coded value. But we might want to add this at a later point when we add the bias, as both influence the route scoring in a similar way.

**Comment by palango** [comment]
> But I think it's best to skip that for now and just use a hard coded value.

I agree, in the future we probably want to make this configurable, but for now we don't need to spend time on this.
